ibm/resource_ibm_iam_access_group_members.go

```diff
@@ -229,9 +233,11 @@ func resourceIBMIAMAccessGroupMembersUpdate(context context.Context, d *schema.R
 		if err != nil {
 			return diag.FromErr(err)
 		}
-		request := prepareMemberAddRequest(userids, serviceids)
+		members := prepareMemberAddRequest(iamAccessGroupsClient, userids, serviceids)
 
-		_, err = iamuumClient.AccessGroupMember().Add(grpID, request)
+		addMembersToAccessGroupOptions := iamAccessGroupsClient.NewAddMembersToAccessGroupOptions(grpID)
+		addMembersToAccessGroupOptions.SetMembers(members)
+		_, _, err = iamAccessGroupsClient.AddMembersToAccessGroup(addMembersToAccessGroupOptions)
 		if err != nil {
 			return diag.FromErr(err)
```

Comment:
return diag.FromErr(err) ===> return diag.FromErr(fmt.Errorf("Error updating members to group(%s). API responce: %s", grpID, detailResponse))